charts/rancher-logging/0.3.1/charts/fluentd/charts/fluentd-windows/values.yaml

```diff
@@ -0,0 +1,52 @@
+nameOverride: fluentd
+labels: {}
+
+image:
+  os: windows
+  repository: rancher/fluentd
+  tag: v0.1.31
```

Comment:
The tag is the only change in this new chart version compared to the previous version. 
In the previous version, the tag is `v0.1.31`